src/templates/settings.html

```diff
@@ -63,7 +63,7 @@
     </div>
   </section>
 
-  <section class="main full hide-on-desktop account-settings-links">
```

Comment:
I think if you're a reviewer you get a link to the reviewer tools in the footer on each page. I think the footer should stay as-is, it was setup that way for a reason. It seems to me like the settings links are just there for when the footer isn't visible so we should hide the settings links instead.